plugins/nf-wave/src/main/io/seqera/wave/plugin/cli/WaveDebugCmd.groovy

```diff
@@ -47,40 +51,73 @@ class WaveDebugCmd {
     }
 
     protected void taskDebug(List<String> args) {
+        if( !args )
+            throw new AbortOperationException("Missing id of the task to debug or remote task path")
+
+        final criteria = args.pop()
+
+        if ( isRemotePath(criteria) ) {
```

Comment:
```suggestion
        if ( isRemotePath(criteria) && args.size()>0 ) {
```

Reply:
Think the desired logic is to perform this only when then args are the remote path followed by the image name

Reply:
I think is better to keep as it is now that fails with a message "Missing container image - usage: nextflow plugin nf-wave:debug-task <remote-task-path> <image>" if the image is not provided.

Reply:
then, the error message can be improved. My point is that when using `s3://some/path` without image, it should work taking the image from the cache as it's doing now

Reply:
How would you improve the message? Maybe...
**"You need to provide the container image to run the process. Usage: nextflow plugin nf-wave:debug-task [remote-task-path] [image]"**

 Not easy to implement to get the container from the remote cache. At S3 we do not have the history file so we cannot known what is the last session ID.

Reply:
This is command for nerds :D I think it's enough 

```
Usage: nextflow plugin nf-wave:debug-task <remote-task-path OR task id> [container image]
```

Reply:
The container is not optional when using remote path.

```
  Local usage: nextflow plugin nf-wave:debug-task <task-path OR task-id>
 Remote usage: nextflow plugin nf-wave:debug-task <remote-task-path> <container-image>
```

Reply:
it does 

```
$ nextflow plugin nf-wave:debug-task s3://nextflow-ci/wave/73/37ddd4ba23e5f20dcf3219dfc0f01e
Launching debug session for task 'RNASEQ:FASTQC (FASTQC on lung)' - work directory: s3://nextflow-ci/wave/73/37ddd4ba23e5f20dcf3219dfc0f01e
2023-02-17_15:43:02 Mounting FusionFS v0.6.1-e6da3a5
(base) root@47871402d2b2:/fusion/s3/nextflow-ci/wave/73/37ddd4ba23e5f20dcf3219dfc0f01e# 
```

Reply:
I went ahead with this